po/fr.po

```diff
@@ -0,0 +1,1428 @@
+"X-Generator: Gtranslator 42.0\n"
+
+#: data/gsettings/ca.andyholmes.Valent.Device.gschema.xml:10
+msgid "Paired"
+msgstr "Lié"
+
+#: data/gsettings/ca.andyholmes.Valent.Device.gschema.xml:11
+msgid "Whether the device is paired"
+msgstr "Si l'appareil est lié"
+
+#: data/gsettings/ca.andyholmes.Valent.gschema.xml:10
+#: src/plugins/runcommand/valent-runcommand-editor.ui:44
+msgid "Name"
+msgstr "Nom"
+
+#: data/gsettings/ca.andyholmes.Valent.gschema.xml:11
+msgid "The display name for the local device."
+msgstr "Le nom d'affichage de l'appareil local"
+
+#: data/gsettings/ca.andyholmes.Valent.Plugin.gschema.xml:10
+msgid "Plugin State"
+msgstr "État de l'extension"
+
+#: data/gsettings/ca.andyholmes.Valent.Plugin.gschema.xml:11
+msgid "Whether the plugin is enabled or not."
+msgstr "Si l'extension est activée ou non"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:11
+#: data/ca.andyholmes.Valent.desktop.in.in:3
+#: src/libvalent/ui/valent-preferences-window.ui:13
+#: src/libvalent/ui/valent-window.c:263 src/libvalent/ui/valent-window.ui:40
+msgid "Valent"
+msgstr "Valent"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:12
+#: data/ca.andyholmes.Valent.desktop.in.in:4
+msgid "Connect, control and sync devices"
+msgstr "Connectez, contrôlez et synchronisez vos appareils"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:15
+msgid ""
+"Securely connect your devices to open files and links where you need them, "
+"get notifications when you need them, stay in control of your media and more."
+msgstr ""
+"Connectez vos appareils en toute sécurité pour ouvrir des fichiers, des "
+"liens, recevez vos notifications sur n'importe quel appareil, et bien plus "
+"encore."
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:19
+msgid "Features:"
+msgstr "Fonctionnalités:"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:23
+msgid "Sync contacts, notifications and clipboard content"
+msgstr ""
+"Synchronisez vos contacts, notification et le contenu de votre presse-papier."
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:24
+msgid "Control media players and volume"
+msgstr "Contrôlez vos lecteurs média et votre volume"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:25
+#: src/plugins/share/share.plugin.desktop.in:8
+msgid "Share files, links and text"
+msgstr "Partagez des fichiers, des liens et du texte"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:26
+msgid "Virtual touchpad and keyboard"
+msgstr "Clavier et pavé tactile virtuel"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:27
+msgid "Call and text notification"
+msgstr "Notifications d'appel et de SMS"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:28
+msgid "Execute custom commands"
+<<<<<<< HEAD
+msgstr "Éxecuter des commandes personnalisées"
+=======
+msgstr "Exécuter des commandes personnalisées"
+>>>>>>> d9df43a (Added french translation)
```

Comment:
```suggestion
<<<<<<< HEAD
msgstr "Éxecuter des commandes personnalisées"
=======
msgstr "Exécuter des commandes personnalisées"
>>>>>>> d9df43a (Added french translation)
```

Looks like this got caught up in a rebase. I'm not sure which is correct though, it's been years :slightly_smiling_face: